packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -107,11 +119,21 @@ export const RichTextEditor: React.VFC<RichTextEditorProps> = props => {
     // Including `onContentChange` in the dependencies here will cause a 'Maximum update depth exceeded' issue
   }, [editorState])
 
+  const validationMessageAria = validationMessage
+    ? `${editorId}-rte-validation-message`
+    : ""
+  const descriptionAria = description ? `${editorId}-rte-description` : ""
+
+  const ariaDescribedBy = [validationMessageAria, descriptionAria].reduce(
+    (prev, curr) => (curr ? [curr, prev].join(" ") : prev),
+    ""
+  )
```

Comment:
```suggestion
  const validationMessageId = validationMessage
    ? `${editorId}-rte-validation-message`
    : ""
  const descriptionId = description ? `${editorId}-rte-description` : ""

  const ariaDescribedBy = `${validationMessageAria} ${descriptionAria}`
```

I know the last one is trying to strip out unnecessary white space, but it doesn't seem to actually do it properly (when only one exists, an extra space is appended). Might be worth having a util at some point, but for the meantime it should be fine with potentially `" "` (unless @dougmacknz tells me it's not allowed).

Reply:
Thanks, just copied it from TextField in case there is something that I am not aware of. will update!